schema-guide.md

```diff
@@ -3160,13 +3160,13 @@ licenses, it is assumed their relationship is OR, not AND.
         type: generic
     ```
 
-Note that this key should contain notes that may be picked up by some downstream tooling (e.g., reference managers), but not other (e.g., software index).
+Note that the notes provided under this key may be picked up by some downstream tooling (e.g., reference managers), but not others (e.g., a software index).
```

Comment:
maybe move this to `description`

Reply:
Fixed in https://github.com/citation-file-format/citation-file-format/pull/285/commits/11ea8a32892ed4bb4b7ebea33f39aa430ee8c750.